ls1intum/Artemis
# `Programming exercises:` Refine code editor refresh-files tooltip and translations

### Checklist
#### General



#### Client

### Motivation and Context



### Description


### Steps for Testing


1. Create a programming exercise with enabled code editor
2. Participate in the exercise and create a merge conflict from the local working environment
3. Check that the translations are appropriate

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots
